CONTRIBUTING.md

```diff
@@ -0,0 +1,190 @@
+# Contributing
+
+This document provides guidelines for contributing to this repository.
+
+## Development Environment Setup
+
+### Prerequisites
+* Basic tools: `curl`, `jq`, `oc`
+* go: v1.19
+* Container engine: `podman`
+
+## Running a test environment
+
+### Kwok Container
```

Comment:
this section seems to be out of data - can you update it for building using our own custom image?

Reply:
Done